src/main/scala/com/netflix/edda/mongo/MongoDatastore.scala

```diff
@@ -69,18 +69,27 @@ object MongoDatastore {
     import collection.JavaConverters._
     obj match {
       case o: BasicDBObject => {
-        o.keySet.asScala.map((key: String) => (key -> mongoToScala(o.get(key)))).toMap
+        o.keySet.asScala.map((key: String) => (mongoDecodeString(key) -> mongoToScala(o.get(key)))).toMap
       }
       case o: BasicDBList => {
         List.empty[Any] ++ o.asScala.map(mongoToScala(_))
       }
       case o: Date => new DateTime(o)
+      case o: String => mongoDecodeString(o)
       case o: AnyRef => o
       case null => null
       case other => throw new java.lang.RuntimeException("mongoToScala: don't know how to handle: " + other)
     }
   }
 
+  def mongoDecodeString(str: String): String = {
+    str.replace("\\0xFF0E", ".").replace("0xFF0E", ".")
```

Comment:
I was attempting to follow the mongo recommendation on the replacement, and I intended to add the $ as well once I got it working, then forgot about it later.  I agree on the \uFF0E.  I'll get a working change and update the PR.

Docs for reference:
https://docs.mongodb.com/v3.0/faq/developers/#dollar-sign-operator-escaping